lnbits/templates/base.html

```diff
@@ -325,9 +285,22 @@
     <script type="text/javascript">
       const themes = {{ LNBITS_THEME_OPTIONS  | tojson }}
       const LNBITS_DENOMINATION = {{ LNBITS_DENOMINATION   | tojson}}
-      if(themes && themes.length) {
+      if (themes && themes.length) {
         window.allowedThemes = themes.map(str => str.trim())
       }
+
+      window.langs = [
+        { value: 'en', label: 'English', display: 'EN' },
+        { value: 'de', label: 'Deutsch', display: 'DE' },
+        { value: 'es', label: 'Español', display: 'ES' },
+        { value: 'jp', label: '日本語', display: 'JP' },
+        { value: 'fr', label: 'Français', display: 'FR' },
+        { value: 'it', label: 'Italiano', display: 'IT' },
+        { value: 'pi', label: 'Pirate', display: 'PI' },
+        { value: 'nl', label: 'Nederlands', display: 'NL' },
+        { value: 'we', label: 'Welsh' },
```

Comment:
Why no `display` field for `Polish` and `Pirate`

Reply:
`Welsh` isn't a `Pirate` language afaik, but I could be wrong ;)

Reply:
@motorina0 , it's an optional field, i don't know, and copilot didn't step in, how to spell `Polish` in ... polish!

@callebtc ??

Reply:
Polski

Reply:
Welsh should be CY, yep, please change